### PR TITLE
[CLI-1040] Deprecate MobileWeb platform master

### PR DESCRIPTION
**Pull request by feons**
https://jira.appcelerator.org/browse/CLI-1040

### CODE REVIEW

File: cli/lib/creator.js [hunk 1]
```diff
@@ -339,6 +339,10 @@ Creator.prototype.configOptionPlatforms = function configOptionPlatforms(order)
 			});
 		}
 
+		if (goodValues.mobileweb) {
+			logger.warn(__('MobileWeb platform has been deprecated.'));
```
Comment (hansemannn) [inline cli/lib/creator.js]: Can you mention 5.4.0 and 7.0.0? :-) Like: "MobileWeb platform has been deprecated in 5.4.0 and will be removed in 7.0.0".

**Comment by hansemannn** [comment]
This should also go into 5.4.0 before we forget it.

**Comment by hansemannn** [comment]
@feons This should also go into 5.4.0 before we forget it. 😊

**Comment by cheekiatng** [comment]
APPROVED.

**Pull request merged**